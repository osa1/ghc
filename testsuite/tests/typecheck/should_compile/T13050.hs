module HolesInfix where

f, g :: Int -> Int -> Int
f x y = _ x y
g x y = x `_` y
