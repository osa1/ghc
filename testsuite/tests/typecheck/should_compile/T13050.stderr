
T13050.hs:4:9: warning: [-Wtyped-holes (in -Wdefault)]
    • Found hole: _ :: Int -> Int -> Int
    • In the expression: _
      In the expression: _ x y
      In an equation for ‘f’: f x y = _ x y
    • Relevant bindings include
        y :: Int (bound at T13050.hs:4:5)
        x :: Int (bound at T13050.hs:4:3)
        f :: Int -> Int -> Int (bound at T13050.hs:4:1)

T13050.hs:5:12: warning: [-Wtyped-holes (in -Wdefault)]
    • Found hole: _ :: Int -> Int -> Int
    • In the expression: _
      In the expression: _ x y
      In an equation for ‘g’: g x y = _ x y
    • Relevant bindings include
        y :: Int (bound at T13050.hs:5:5)
        x :: Int (bound at T13050.hs:5:3)
        g :: Int -> Int -> Int (bound at T13050.hs:5:1)
